Dockerfile

```diff
@@ -151,6 +150,7 @@ RUN --mount=from=containerd-src,src=/usr/src/containerd,readwrite --mount=target
 # containerd v1.4 for integration tests
 FROM containerd-base as containerd-alt
 ARG CONTAINERD_ALT_VERSION
+ARG GO111MODULE=off
```

Comment:
TBH, not sure why go1.16 didn't detect vendor here, and complains about no `go.mod` being present; guess it still requires a `go.mod` even if a `vendor` is present 🤷‍♂️ 

I'lll have a look at adding this to the containerd makefile in the 1.4 branch

Or would it be better to build this version of containerd with Go 1.15? https://github.com/containerd/containerd/blob/v1.4.6/.github/workflows/ci.yml#L29

Reply:
It doesn't matter until the tests pass. These binaries are never released.